docs/permissions/permission_use_cases.md

```diff
@@ -41,6 +41,13 @@ To create and publish content, users must additionally have the following Polici
 
 This also lets the user copy and move content, as well as add new Locations to a Content item (but not remove them).
 
+## Move content
+
+To move content to another Location or move the Subtree, the user must have the following Policy:
```

Comment:
Technically I would say there is no 'move content' - you are always moving a Subtree (which might only have one Node), so:

```To move a Subtree, the user must have the following Policies```

But the terminology 'move content' is used at several other places -in flowing text usually 'Content Item' or 'Content Items'  is used instead of 'content' (this would apply to the whole text). So my best guess is:

```To move a Content item or a Subtree, the user must have the following Policies```